tests/testProjectClassToR1.m

```diff
@@ -0,0 +1,286 @@
+classdef testProjectClassToR1 < matlab.unittest.TestCase
+
+    properties
+        inputStandardProjectClass
+        inputStandardStruct
+        outputStandardStruct
+        defaultProject
+        outputCustomStruct
+        inputCustomProjectClass
+        modelFile
+        fullPath
+    end
+
+    methods(TestClassSetup)
+        function defineVariables(testCase)
+            testCase.inputStandardProjectClass = "originalDSPCBilayerProjectClass.mat";
+            testCase.inputStandardStruct = "originalDSPCBilayerStructInput.mat";
+            testCase.outputStandardStruct = "originalDSPCBilayerStructOutput.mat";
+            testCase.defaultProject = "defaultProject.mat";
+            testCase.outputCustomStruct = "monolayerVolumeModel.mat";
+            testCase.inputCustomProjectClass = "monolayerVolumeModelProjectClass.mat";
+            testCase.modelFile = "monolayerVolumeCustomLayerModel.m";
+        end
+    end
+
+    methods(TestMethodSetup)
+        function setWorkingFolder(testCase)
+            root = getappdata(0,'root');
+            testCase.fullPath = [root filesep 'tests' filesep 'testProjectClassToR1Conversion'];
+            import matlab.unittest.fixtures.CurrentFolderFixture
+            testCase.applyFixture(CurrentFolderFixture(testCase.fullPath))
+        end
+    end 
+
+     methods(TestMethodTeardown)
+        function testCleanUp(testCase)
+            cd (testCase.fullPath)
+            if exist('newDirectory','dir')
+                rmdir('newDirectory', 's')
+            end
+        end
+    end 
+
+    methods (Test)
+
+        function testFilesExists(testCase)
+            testCase.verifyEqual(exist(testCase.outputStandardStruct,'file'), 2);
```

Comment:
I don't think this is a useful test, it is not testing any function. it is okay to assume test data is always present

Reply:
I added these tests because of the issue with the last pull request. I deleted a file that was being used. That is why I wanted to write such tests so that if someone accidentally deletes a file from the folder, then this test will fail. But I can remove these test for now.